index.html

```diff
@@ -67,7 +88,25 @@
 
         ga('create', gaID, 'auto');
         ga('send', 'pageview');
+  //Scroll to the top of the page.
+        window.onscroll = function() {scrollFunction()};
+
+        function scrollFunction() {
+        if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+        document.getElementById("myBtn").style.display = "block";
+        } else {
+        document.getElementById("myBtn").style.display = "none";
+         }
+         }
+
+  // When the user clicks on the button, scroll to the top of the document
+       function topFunction() {
+       document.body.scrollTop = 0; // For Safari
+       document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
+       }
       </script>
 
+    <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
```

Comment:
This file shouldn't contain any app logic. I think this code is better placed at the [SearchGrid component](https://github.com/creativecommons/cccatalog-frontend/blob/master/src/components/SearchGrid.vue)

Reply:
It means we have to change this on SearchGrid.vue file to enable this scroll while searching??

Reply:
Yes. That component is where this code would be best placed I think

Reply:
Yes I am trying to place this code on Searchgrid.vue